scripts/e2e/compare_metrics.py

```diff
@@ -0,0 +1,70 @@
+# Copyright (c) 2024 The Jaeger Authors.
+# SPDX-License-Identifier: Apache-2.0
+
+import argparse
+import difflib
+import sys
+from pathlib import Path
+
+def read_metric_file(file_path):
+    try:
+        with open(file_path, 'r') as f:
+            return f.readlines()
+    except FileNotFoundError:
+        print(f"Error: File not found - {file_path}")
+        sys.exit(1)
+    except Exception as e:
+        print(f"Error reading file {file_path}: {str(e)}")
+        sys.exit(1)
+
+def generate_diff(file1_lines, file2_lines, file1_name, file2_name):
```

Comment:
this is incorrect. A metric is in the form
```
name{label1=v1, ...} value
```
We don't care about `value` (it can be different in different runs). We do care about name and labels, but not about the order of the labels.